docs/intro.md

```diff
@@ -42,29 +44,44 @@ First, install [Rust][rust].
 This should automatically install `cargo`.
 
 Clone the repository:
+
 ```
 git clone https://github.com/calyxir/calyx.git
 ```
+
 Then build the compiler:
+
 ```
 cargo build
 ```
 
 You can build and run the compiler with:
+
 ```
 cargo build # Builds the compiler
 ./target/debug/calyx --help # Executes the compiler binary
 ```
 
 We recommend installing the git hooks to run linting and formatting checks before each commit:
+
 ```shell
 /bin/sh setup_hooks.sh
 ```
 
+You can build the docs by installing `mdbook` and the callouts preprocessor:
+
+```sh
+cargo install mdbook 
+cargo install --git https://github.com/ToolmanP/rs-mdbook-callout
```

Comment:
Any way to install this in a versioned manner? I'm a little concerned about installing from github.